packages/trader/src/Modules/Trading/Containers/trade.jsx

```diff
@@ -147,12 +148,17 @@ const Trade = ({
     );
 
     const form_wrapper_class = isMobile() ? 'mobile-wrapper' : 'sidebar__container desktop-only';
+    const chart_height_offset = React.useMemo(() => {
+        if (is_accumulator) return '295px';
```

Comment:
what if both of these conditions were **true** :thinking:

Reply:
they cannot be true at the same time, only 1 of them is true at every point in time🙂